permutect/parameters.py

```diff
@@ -117,6 +120,8 @@ def add_training_params_to_parser(parser):
                         help='number of epochs for primary training loop')
     parser.add_argument('--' + constants.NUM_CALIBRATION_EPOCHS_NAME, type=int, default=0, required=False,
                         help='number of calibration-only epochs')
+    parser.add_argument('--' + constants.INFERENCE_BATCH_SIZE_NAME, type=int, default=8192, required=False,
```

Comment:
This will also need to go in the WDL scripts, but I'll handle that in a follow-up PR.